Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -2182,103 +1610,6 @@ public String removePages() {
         return null;
     }
 
-    /**
-     * OCR.
-     */
-    public boolean isShowOcrButton() {
-        return ConfigCore.getBooleanParameterOrDefaultValue(ParameterCore.SHOW_OCR_BUTTON);
-    }
-
-    /**
-     * Show OCR result.
-     */
-    public void showOcrResult() {
```

Comment:
Will this functionality (showing OCR results) added back in future?

Reply:
Yes